src/Shared/DiagnosticIds/DiagnosticIds.cs

```diff
@@ -111,6 +111,12 @@ internal static class Metrics
         internal const string METGEN018 = nameof(METGEN018);
         internal const string METGEN019 = nameof(METGEN019);
     }
+
+    internal static class AuditReports
+    {
+        internal const string AUDREP000 = nameof(AUDREP000);
+        internal const string AUDREP001 = nameof(AUDREP001);
```

Comment:
Do we need to document these in https://github.com/dotnet/extensions/blob/main/docs/list-of-diagnostics.md

Reply:
Done both: https://aka.ms/dotnet-extensions-warnings/AUDREPGEN001